assembly/__tests__/i128.spec.ts

```diff
@@ -90,6 +91,85 @@ describe("Basic Operations", () => {
     let neg = -one;
     expect(i128.abs(neg)).toStrictEqual(one);
   });
+  it("Should left shift one number", () => {
+    var a = new i128(1, 0);
+    expect(a << 65).toStrictEqual(new i128(0, 2));
+  });
+
+  it("Should periodic left shift one number", () => {
+    var a = new i128(1, 0);
+    expect(a << (65 + 128)).toStrictEqual(new i128(0, 2));
+  });
+
+  it("Should invariant left shift zero number", () => {
+    var a = new i128(1, 1);
+    expect(a << 0).toStrictEqual(a);
+  });
+
+  it("Should right shift one number", () => {
+    var a = new i128(0, 100);
+    expect(a >> 65).toStrictEqual(new i128(50));
+  });
```

Comment:
I'm not sure `>>` op for `i128` is correct. Please add tests for negative values, for value without hi or lo parts and shift lenght: `16`, `32`, `63`, `64`, `127`

Reply:
you're right, not all tests pass for neg value, I will try to fix this when I have some time